src/main/java/net/dv8tion/jda/api/hooks/AnnotatedEventManager.java

```diff
@@ -55,11 +55,12 @@
     private final Map<Class<?>, Map<Object, List<Method>>> methods = new ConcurrentHashMap<>();
 
     @Override
-    public void register(@Nonnull Object listener)
+    public void register(@Nonnull Object...pListeners)
```

Comment:
This changes the method signature in a breaking way. The ABI method signature becomes an array which means all code referencing this method must be recompiled, as such this is breaking.

Reply:
I'll edit it so that it doesn't change the method signature.